### DISCUSSION TITLE
Query remote json file

### DISCUSSION THREAD

**Discussion by raspharaoh** [noborus/trdsql#279] [Q&A]
Is it possible to query a remote **json** file?
For example:
`trdsql -ijson -oraw -db mdb 'SELECT id, full_name FROM https://api.github.com/repos/noborus/trdsql'
`

**Comment by noborus** [comment]
Thank you for your question. 
Now that's not possible.
This is an interesting feature, but I won't implement it in `trdsql` itself.
